packages/astro/e2e/multiple-frameworks.test.js

```diff
@@ -14,89 +14,89 @@ test.afterAll(async () => {
 });
 
 test.describe('Multiple frameworks', () => {
-	test('React counter', async ({ astro, page }) => {
+	test.skip('React counter', async ({ page }) => {
```

Comment:
This is the one test we actually need to skip. What seems to be happening is:

- The initial page loads too slowly.
- We change a file.
- The change isn't reflected because the page hasn't finished yet.

I tried fixing this by making us wait on the initial load better, but it did not work. Note that this only fails on Windows ~50% of the time. Seems like skipping it to getting CI consistency is good for now.